package.json

```diff
@@ -38,35 +38,35 @@
     "node": ">=6.11.5"
   },
   "dependencies": {
-    "@babel/cli": "^7.6.3",
-    "@babel/core": "^7.6.3",
-    "acorn": "^6.3.0",
+    "@babel/cli": "^7.7.0",
+    "@babel/core": "^7.7.2",
+    "acorn": "^7.1.0",
     "babel-loader": "^8.0.6",
-    "chalk": "^2.4.2",
+    "chalk": "^3.0.0",
     "circular-dependency-plugin": "^5.2.0",
-    "copy-template-dir": "1.4.0",
-    "cross-spawn": "6.0.5",
-    "eslint": "^6.5.1",
-    "inquirer": "^6.5.2",
+    "copy-template-dir": "^1.4.0",
+    "cross-spawn": "^7.0.1",
+    "eslint": "^6.6.0",
+    "inquirer": "^7.0.0",
     "jest": "^24.9.0",
-    "ora": "^3.4.0",
-    "prettier": "^1.18.2",
-    "react-styleguidist": "^9.1.16",
+    "ora": "^4.0.3",
+    "prettier": "^1.19.1",
+    "react-styleguidist": "^10.2.1",
     "resolve": "^1.12.0",
     "resolve-pkg": "^2.0.0",
-    "rimraf": "^2.7.1",
-    "run-parallel": "1.1.9",
-    "run-series": "1.1.8",
-    "webpack": "^4.41.0",
-    "yargs": "^13.3.0"
+    "rimraf": "^3.0.0",
+    "run-parallel": "^1.1.9",
+    "run-series": "^1.1.8",
+    "webpack": "^4.41.2",
+    "yargs": "^15.0.1"
   },
   "devDependencies": {
     "babel-preset-zillow": "^4.2.0",
-    "eslint-plugin-zillow": "^3.4.1",
-    "husky": "^3.0.8",
-    "react": "^16.10.2",
-    "react-dom": "^16.10.2",
-    "standard-version": "^6.0.1",
+    "eslint-plugin-zillow": "^3.5.1",
+    "husky": "^3.1.0",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "standard-version": "^7.0.1",
```

Comment:
I'm wary about upgrading this -- the last few times I have tried it produced unexpected results when creating the CHANGELOG.

Reply:
Do we have pointers on specific problems? Would it be worth filing issues to `standard-version` for specific behaviour?

Reply:
It's been a while, but maybe the issue was just this:
https://github.com/conventional-changelog/standard-version/issues/424